<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBox1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-9999999</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>/</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>r√≥w kwadr</string>
     </property>
    </item>
   </widget>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99999</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBox1_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
